From 61ce13e4ab846aa035037217c5eec6aff229e539 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 28 Jan 2019 14:42:34 +0000
Subject: [PATCH] input: rpi-ft5406: Clear build warning on 64 bit
 builds.

Resolve 64 bit build warning over using %x with a dma_addr_t.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/input/touchscreen/rpi-ft5406.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/input/touchscreen/rpi-ft5406.c
+++ b/drivers/input/touchscreen/rpi-ft5406.c
@@ -218,8 +218,8 @@ static int ft5406_probe(struct platform_
 
 	if (!ts->ts_base) {
 		dev_warn(dev,
-		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%x)\n",
-		err, touchbuf, ts->ts_base, ts->bus_addr);
+		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%pad)\n",
+		err, touchbuf, ts->ts_base, &ts->bus_addr);
 
 		err = rpi_firmware_property(
 				fw,
